src/core/proxy.js

```diff
@@ -35,7 +40,10 @@ const requestHandler = async (request, proxy, overrides = {}) => {
             headers: response.headers,
             body: response.body
         });
-    } catch(error) {await request.abort()}
+    } catch (error) {
+        await request.abort();
+        throw error;
+    }
```

Comment:
Use `} catch(error) {await request.abort()}` instead (?)

Reply:
Ah I see. I'll revert it then.